addons/survey/tests/test_survey_ui_feedback.py

```diff
@@ -164,6 +164,79 @@ def test_03_public_survey_tour(self):
         access_token = self.survey_feedback.access_token
         self.start_tour("/survey/start/%s" % access_token, 'test_survey')
 
+    def test_04_public_survey_with_triggers(self):
```

Comment:
Would be great to add a small docstring with a schema of the questions, and what it tests. Otherwise it is not obvious when you have a lot of tests to know what they really test :)